src/components/DateTimePicker/index.js

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable react/prop-types */
+import React, { useRef, useState, useImperativeHandle } from 'react';
+import PropTypes from 'prop-types';
+import withReduxForm from '../../libs/hocs/withReduxForm';
+import Input from '../DatePicker/input';
+import CalendarIcon from '../DatePicker/calendarIcon';
+import DateTimePickerModal from './pickerModal';
+import formatDateTime from './helpers/formatDateTime';
+import { ENTER_KEY, SPACE_KEY } from '../../libs/constants';
+import Styled from './styledComponents';
```

Comment:
for styled components let's put them in a styled folder and make one file per styled component as:
/styled
    /container.js
    /header.js
    ...